Add link for Gallery and Playground on Docs Reference

rafael-rosa-knowcode opened the pull request:
GitHub Issue (If applicable): closes #

https://github.com/unoplatform/uno/issues/14746

## PR Type

What kind of change does this PR introduce?

- Documentation content changes

## What is the current behavior?

We do not have link for Gallery or Playground in the Docs - [Implemented Views](https://platform.uno/docs/articles/implemented-views.html)

## What is the new behavior?

We have some links to Gallery and Playground

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below.
     Please note that breaking changes are likely to be rejected -->

## Other information

<!-- Please provide any additional information if necessary -->

Internal Issue (If applicable):
<!-- Link to relevant internal issue if applicable. All PRs should be associated with an issue (GitHub issue or internal) -->

unodevops commented:
🤖 Your WebAssembly Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-14833/index.html

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-14833/index.html

jeromelaban commented on src/Uno.UWPSyncGenerator/DocGenerator.cs:
```diff
@@ -140,6 +140,18 @@ void BuildMemberLists()
+								var playgroundLink = GetPlaygroundLink(viewName);
+								if (playgroundLink != null)
+								{
+									_sb.AppendParagraph($"To better understand how {formattedViewName} works, you can use the Gallery {Hyperlink("here", playgroundLink)}.");
```
This is working better now: https://unodocsprstaging.z13.web.core.windows.net/pr-14833/articles/implemented/microsoft-ui-xaml-controls-combobox.html, We need to adjust the text now. It's probably best to merge the two to end up like this: `To better understand how XX works, you can use the [Uno Gallery] or the [Uno Playground]`. A string join with ` or ` should do the trick.

rafael-rosa-knowcode replied:
I was already forwarding an update to look like this.

	_To better understand how XX works, you can use the Gallery here.
	Or run your own tests on XX here on Playground._

Would it look better in one line?

Had I thought about passing it on to Ludvig to think of a way to make the change more prominent?
Or suddenly create a topic for it.
It can help users a lot to be able to look at the rules (microsoft link) and easily see how it works in the gallery and test it in the playground.

Do you think it's worth it or is that ok?

rafael-rosa-knowcode replied:
![image](https://github.com/unoplatform/uno/assets/116665025/c9e99387-7468-4bb2-8964-274ad84db117)

rafael-rosa-knowcode replied:
@jeromelaban 

The way the image/link bellow looks is it good or should I change it to just one line?

![image](https://github.com/unoplatform/uno/assets/116665025/d1f976d5-2683-4d1c-8c1b-2876d0d4160a)

e.g https://unodocsprstaging.z13.web.core.windows.net/pr-14833/articles/implemented/microsoft-ui-xaml-controls-combobox.html

rafael-rosa-knowcode commented:
This issue need to wait for the update on the Gallery. That allow the link redirection.

https://github.com/unoplatform/Uno.Gallery/pull/1054

unodevops commented:
🤖 Your WebAssembly Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-14833/index.html

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-14833/index.html

jeromelaban commented on src/Uno.UWPSyncGenerator/DocGenerator.cs:
```diff
@@ -140,6 +140,18 @@ void BuildMemberLists()
+									_sb.AppendParagraph($"To better understand how {formattedViewName} works, you can use the Gallery {Hyperlink("here", galleryLink)}.");
+								}
+
+								var playgroundLink = GetPlaygroundLink(viewName);
+								if (playgroundLink != null)
+								{
+									_sb.AppendParagraph($"Or run your own tests on {formattedViewName} {Hyperlink("here", playgroundLink)} on {Hyperlink("Playground", playgroundLink)}.");
+								}
```
If there's only the playground link, this will look like an odd sentence... :) You'll need to merge both sentences conditional to `galleryLink` and `playgroundLink`.

rafael-rosa-knowcode replied:
@jeromelaban 

But, the idea is to have links to the Playground and the Gallery.
To add the functionality to the Gallery, I made a PR that allows the link to be redirected to the specific Sample when receiving a hash. https://github.com/unoplatform/Uno.Gallery/pull/1054

And with that, I left two lines here. 
This would emphasize and highlight both features.

jeromelaban replied:
Yes it is better to unify both links, using something like this:

```suggestion
								var galleryLink = GetGalleryLink(viewName);
								var playgroundLink = GetPlaygroundLink(viewName);
								
								if (galleryLink is not null or playgroundLink is not null)
								{
								    Lists<string> options = new();
								    
								    if(galleryLink is not null)
								    {
								       options .Add($"use the Uno Gallery {Hyperlink("here", galleryLink)}");
								    }			
								    	    
								    if(playgroundLink is not null)
								    {
								       options .Add($"run your own tests on {formattedViewName} {Hyperlink("here", playgroundLink)} on {Hyperlink("Uno Playground", playgroundLink)}");
								    }
								    
								    _sb.AppendParagraph($"To better understand how {formattedViewName} works, you can" + string.Join("or", options) + ".");
								}
```

unodevops commented:
🤖 Your WebAssembly Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-14833/index.html

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-14833/index.html

rafael-rosa-knowcode commented:
@jeromelaban 

I changed the code a little.
If you can validate.
I removed the "here" and included the link in the words Uno Gallery and Uno Platform

Link: https://unodocsprstaging.z13.web.core.windows.net/pr-14833/articles/implemented/microsoft-ui-xaml-controls-combobox.html

Image.

![image](https://github.com/unoplatform/uno/assets/116665025/79e5c08b-a140-4177-b867-4a87f84a365e)

And i reverted the commandLineArgs to sync.

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-14833/index.html

nventive-devops commented:
The [build 106080](https://dev.azure.com/uno-platform/Uno%20Platform/_build/results?buildId=106080) found UI Test snapshots differences: `android-28-net7`: **0**, `android-28-net7-Snap`: **47**, `ios`: **39**, `ios-Snap`: **666**, `skia-gtk-screenshots`: **21**, `skia-wpf-screenshots`: **18**, `wasm`: **32**, `wasm-automated-net7.0-UWP-Benchmarks`: **0**, `wasm-automated-net7.0-UWP-Default`: **31**, `wasm-automated-net7.0-UWP-RuntimeTests`: **0**, `wasm-automated-net7.0-WinUI-RuntimeTests`: **0**

<details>
<summary>Details</summary>


* `android-28-net7`: **0** changed over 1

* `android-28-net7-Snap`: **47** changed over 1049

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `Grid_Grid_with_UserControl_VerticalAlignment_Variable_Width_Uno_UI_Samples_Content_UITests_GridTestsControl_Grid_with_UserControl_VerticalAlignment_Variable_Width`
  - `Grid_Quadrant_absolute_split_Uno_UI_Samples_Content_UITests_GridTestsControl_Quadrant_absolute_split`
  - `Pointers_Touch_Uno_UI_Samples_Content_UITests_TouchEventsTests_Touch`
  - `RatingControl_UITests_Microsoft_UI_Xaml_Controls_RatingControlTests_RatingControlPage_UITests_Microsoft_UI_Xaml_Controls_RatingControlTests_RatingControlPage`
  - `TabView_UITests_Microsoft_UI_Xaml_Controls_TabViewTests_TabViewPage_UITests_Microsoft_UI_Xaml_Controls_TabViewTests_TabViewPage`
  - `BreadcrumbBar_MUXControlsTestApp_BreadcrumbBarPage_MUXControlsTestApp_BreadcrumbBarPage`
  - `Control_BasicEnability_UITests_Shared_Windows_UI_Xaml_Enability_BasicEnability`
  - `ListView_HorizontalListView_Templates_SamplesApp_Windows_UI_Xaml_Controls_ListView_HorizontalListView_Templates`
  - `ListView_ListViewLargeLegacy_SamplesApp_Windows_UI_Xaml_Controls_ListView_ListViewLargeLegacy`
  - `ListView_ListView_Changing_Text_SamplesApp_Windows_UI_Xaml_Controls_ListView_ListView_Changing_Text`
  - `MediaPlayerElement_Mini_player_UITests_Shared_Windows_UI_Xaml_Controls_MediaPlayerElement_MediaPlayerElement_Minimal`
  - `TextBox_TextBox_Padding_In_ScrollViewer_Uno_UI_Samples_Content_UITests_TextBoxControl_TextBox_Padding_In_ScrollViewer`
  - `TextBox_Uno_UI_Samples_Content_UITests_TextBoxControl_TextBox_In_ContentDialog_Uno_UI_Samples_Content_UITests_TextBoxControl_TextBox_In_ContentDialog`
  - `MediaPlayerElement_Using_avi_UITests_Shared_Windows_UI_Xaml_Controls_MediaPlayerElement_MediaPlayerElement_Avi_Extension`
  - `Expander_WinUIExpanderPage_UITests_Shared_Microsoft_UI_Xaml_Controls_ExpanderTests_WinUIExpanderPage`
  - `Gesture_Recognizer_Pointer_Events_test_bench_UITests_Shared_Windows_UI_Input_GestureRecognizer_PointersEvents`
  - `Grid_Simpletwocolumnsplitgrid_Uno_UI_Samples_Content_UITests_GridTestsControl_Simpletwocolumnsplitgrid`
  - `SwipeControl_MUXControlsTestApp_SwipeControlPage2_MUXControlsTestApp_SwipeControlPage2`
  - `UIElement_UITests_Windows_UI_Xaml_UIElementTests_UIElement_BringIntoView_Nested_ScrollViewers_UITests_Windows_UI_Xaml_UIElementTests_UIElement_BringIntoView_Nested_ScrollViewers`
  - `UIElement_UITests_Windows_UI_Xaml_UIElementTests_UIElement_InitializationSequence_UITests_Windows_UI_Xaml_UIElementTests_UIElement_InitializationSequence`
  </details>

* `ios`: **39** changed over 266

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `RunRuntimeTests_SamplesApp_Samples_UnitTests_UnitTestsPage`
  - `Screenshots_Image_Stretch_Alignment_Wider_Mode-0`
  - `ListView_ObservableCollection_Creation_Count_Added_above`
  - `ListView_ObservableCollection_Creation_Count_Removed_above`
  - `Screenshots_Image_Stretch_Alignment_Wider_Mode-3`
  - `When_ExtendedSelectionWithoutItemClick_Then_PointersEvents_UITests_Windows_UI_Xaml_Controls_ListView_ListView_Selection_Pointers`
  - `ScrollViewer_WhenSync_RunNormalAndCompletesWithNonIntermediate_UITests_Shared_Windows_UI_Xaml_Controls_ScrollViewerTests_ScrollViewer_UpdatesMode`
  - `TextBox_TextChanged_UITests_Shared_Windows_UI_Xaml_Controls_TextBoxTests_TextBox_TextChanged`
  - `When_MultipleSelectionWithoutItemClick_Then_PointersEvents_UITests_Windows_UI_Xaml_Controls_ListView_ListView_Selection_Pointers`
  - `ComboBoxTests_ToggleDisabled_ComboBox_Disabled`
  - `Flyout_ShowAt_Window_Content_UITests_Windows_UI_Xaml_Controls_FlyoutTests_Flyout_ShowAt_Window_Content`
  - `ComboBoxTests_ToggleDisabled_ComboBox_Enabled`
  - `Keyboard_Textbox_NoScrollViewer_Validation_1_-_Remove_Focus_on_filledTextBox`
  - `ListView_ObservableCollection_Creation_Count_Added`
  - `ListView_ObservableCollection_Creation_Count_Scrolled1`
  - `ListView_ObservableCollection_Creation_Count_Scrolled2`
  - `Screenshots_Image_Stretch_Alignment_Equal_Mode-3`
  - `ProgressRing_IsEnabled_Running_UITests_Windows_UI_Xaml_Controls_ProgressRing_WindowsProgressRing_GH1220`
  - `Screenshots_Image_Stretch_Alignment_Wider_Mode-1`
  - `ScrollViewer_WhenAsync_RunIdleAndCompletesWithNonIntermediate_UITests_Shared_Windows_UI_Xaml_Controls_ScrollViewerTests_ScrollViewer_UpdatesMode`
  </details>

* `ios-Snap`: **666** changed over 1040

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `Animations_UITests_Windows_UI_Xaml_Media_Animation_ColorAnimation_Fill_UITests_Windows_UI_Xaml_Media_Animation_ColorAnimation_Fill`
  - `Border_BorderCenteredwithmargins_Uno_UI_Samples_UITests_BorderTestsControl_BorderCenteredwithmargins`
  - `Border_Border_Simple_No_Background_Uno_UI_Samples_UITests_BorderTestsControl_Border_Simple_No_Background`
  - `Border_UITests_Shared_Windows_UI_Xaml_Controls_BorderTests_Border_CornerRadius_Toggle_UITests_Shared_Windows_UI_Xaml_Controls_BorderTests_Border_CornerRadius_Toggle`
  - `Buttons_Buttons_Native_UITests_Shared_Windows_UI_Xaml_Controls_Buttons_Native`
  - `Buttons_Hyperlink_Disabled_nVentive_Umbrella_Views_UI_Samples_Content_UITests_ButtonTestsControl_Hyperlink_Disabled`
  - `Buttons_UITests_Microsoft_UI_Xaml_Controls_RadioButtonsTests_RadioButtonsPage_UITests_Microsoft_UI_Xaml_Controls_RadioButtonsTests_RadioButtonsPage`
  - `Buttons_UITests_Shared_Windows_UI_Xaml_Controls_RadioButtonTests_RadioButton_IsEnabled_Automated_UITests_Shared_Windows_UI_Xaml_Controls_RadioButtonTests_RadioButton_IsEnabled_Automated`
  - `ComboBox_SamplesApp_Wasm_Windows_UI_Xaml_Controls_ComboBox_ComboBox_Header_SamplesApp_Wasm_Windows_UI_Xaml_Controls_ComboBox_ComboBox_Header`
  - `ComboBox_UITests_Shared_Windows_UI_Xaml_Controls_ComboBox_ComboBox_DropDownPlacement_UITests_Shared_Windows_UI_Xaml_Controls_ComboBox_ComboBox_DropDownPlacement`
  - `Default_UITests_Toolkit_ElevatedView_Levels_UITests_Toolkit_ElevatedView_Levels`
  - `Default_UITests_Toolkit_ElevatedView_Tester_UITests_Toolkit_ElevatedView_Tester`
  - `GridView_Uno_UI_Samples_Content_UITests_GridView_GridViewVariableItemHeight_Uno_UI_Samples_Content_UITests_GridView_GridViewVariableItemHeight`
  - `GridView_Uno_UI_Samples_Content_UITests_GridView_GridView_Inside_ScrollViewer_Uno_UI_Samples_Content_UITests_GridView_GridView_Inside_ScrollViewer`
  - `Grid_Quadrant_absolute_split_Uno_UI_Samples_Content_UITests_GridTestsControl_Quadrant_absolute_split`
  - `Grid_Quadrant_all_100_Uno_UI_Samples_Content_UITests_GridTestsControl_Quadrant_all_100`
  - `Image_ImageWithLateSourceFixedDimensions_UITests_Windows_UI_Xaml_Controls_ImageTests_ImageWithLateSourceFixedDimensions`
  - `Image_ImageWithLateSourceUniformToFill_UITests_Windows_UI_Xaml_Controls_ImageTests_ImageWithLateSourceUniformToFill`
  - `ListView_ListViewContainerFromItem_SamplesApp_Windows_UI_Xaml_Controls_ListView_ListViewContainerFromItem`
  - `ListView_ListViewDynamicOrientation_SamplesApp_Windows_UI_Xaml_Controls_ListView_ListViewDynamicOrientation`
  </details>

* `skia-gtk-screenshots`: **21** changed over 1139

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `Brushes-ImageBrushChangingCornerRadius.png`
  - `DragAndDrop-DragDrop_TestPage.png`
  - `Image-Image_Stretch.png`
  - `MUX-WinUIProgressRingPage.png`
  - `MUX-WinUIProgressRing_CustomSources.png`
  - `PipsPager-PipsPagerExamples.png`
  - `MediaPlayerElement-MediaPlayerElement_Stretch_Fill.png`
  - `Pickers-CalendarView_Theming.png`
  - `ListView-ListViewSelectedItems.png`
  - `TextBlock-TextBoxSizeChanging.png`
  - `Progress-ProgressRingHeightTest.png`
  - `Progress-WindowsProgressRing_GH1220.png`
  - `TextBlock-TextBlockMultilineInStarStackPanel.png`
  - `Windows.UI.Composition-CompositionMaskBrush.png`
  - `Windows.UI.Composition-CompositionVisualSurface.png`
  - `Progress-WinUIProgressRingPage.png`
  - `Progress-WinUIProgressRing_CustomSources.png`
  - `Windows.UI.Composition-RedirectVisual.png`
  - `WinUI-PipsPagerExamples.png`
  - `Image-ImageSourceStream.png`
  </details>

* `skia-wpf-screenshots`: **18** changed over 1139

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `MUX-WinUIProgressRing_CustomSources.png`
  - `ListView-ListViewSelectedItems.png`
  - `Progress-WinUIProgressRingPage.png`
  - `Progress-WinUIProgressRing_CustomSources.png`
  - `Progress-WinUIProgressRing_Features.png`
  - `PipsPager-PipsPagerExamples.png`
  - `DragAndDrop-DragDrop_TestPage.png`
  - `MUX-WinUIProgressRingPage.png`
  - `Pickers-CalendarView_Theming.png`
  - `Progress-ProgressRingHeightTest.png`
  - `Progress-WindowsProgressRing_GH1220.png`
  - `Brushes-ImageBrushChangingCornerRadius.png`
  - `Transform-TransformToVisual_Translate2d.png`
  - `Image-Image_Stretch.png`
  - `WinUI-PipsPagerExamples.png`
  - `Windows.UI.Composition-RedirectVisual.png`
  - `Windows.UI.Composition-CompositionMaskBrush.png`
  - `Windows.UI.Composition-CompositionVisualSurface.png`
  </details>

* `wasm`: **32** changed over 1045

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `MUXControlsTestApp.NavigationViewBlankPage1`
  - `UITests.Shared.Windows_UI_Xaml_Controls.MediaPlayerElement.MediaPlayerElement_Stretch_Fill`
  - `UITests.Windows_UI_ViewManagement.ApplicationViewSizing`
  - `MUXControlsTestApp.PipsPagerExamples`
  - `SamplesApp.Wasm.Windows_UI_Xaml_Controls.ListView.ListView_IsSelected`
  - `MUXControlsTestApp.SwipeControlPage2`
  - `SamplesApp.Samples.Microsoft_UI_Xaml_Controls.NavigationViewTests.FluentStyle.FluentStyle_NavigationViewSample`
  - `UITests.Uno_Web.Http.CookieManagerTests`
  - `UITests.Windows_UI_Xaml_Controls.CalendarView.CalendarView_Theming`
  - `MUXControlsTestApp.NavigationViewCompactPaneLengthTestPage`
  - `SamplesApp.Samples.NavigationViewSample.NavigationViewSample`
  - `UITests.Microsoft_UI_Xaml_Controls.ProgressRing.WinUIProgressRing_Features`
  - `UITests.Microsoft_UI_Xaml_Controls.NavigationViewTests.NavigationViewBasicPage`
  - `UITests.Shared.Microsoft_UI_Xaml_Controls.ColorPickerTests.WinUIColorPickerPage`
  - `UITests.Shared.Windows_UI_Xaml_Controls.MediaPlayerElement.MediaPlayerElement_3gp_Extension`
  - `UITests.Shared.Windows_UI_Xaml_Controls.MediaPlayerElement.MediaPlayerElement_Avi_Extension`
  - `SamplesApp.Samples.Windows_UI_Xaml_Controls.NavigationViewTests.NavigationView_BasicNavigation`
  - `UITests.Shared.Windows_UI_Xaml_Controls.MediaPlayerElement.MediaPlayerElement_Mkv_Extension`
  - `UITests.Shared.Windows_UI_Xaml_Controls.MediaPlayerElement.MediaPlayerElement_Mov_Extension`
  - `UITests.Shared.Windows_UI_Xaml_Controls.MediaPlayerElement.MediaPlayerElement_Mp3_Extension`
  </details>

* `wasm-automated-net7.0-UWP-Benchmarks`: **0** changed over 1

* `wasm-automated-net7.0-UWP-Default`: **31** changed over 1937

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `Default_SamplesApp_Wasm_Windows_UI_Xaml_Controls_ListView_ListView_IsSelected_SamplesApp_Wasm_Windows_UI_Xaml_Controls_ListView_ListView_IsSelected`
  - `ListView_ListViewSelectedItems_SamplesApp_Windows_UI_Xaml_Controls_ListView_ListViewSelectedItems`
  - `ColorPicker_WinUIColorPickerPage_UITests_Shared_Microsoft_UI_Xaml_Controls_ColorPickerTests_WinUIColorPickerPage`
  - `Pickers_UITests_Windows_UI_Xaml_Controls_CalendarView_CalendarView_Theming_UITests_Windows_UI_Xaml_Controls_CalendarView_CalendarView_Theming`
  - `MediaPlayerElement_Sources_UITests_Shared_Windows_UI_Xaml_Controls_MediaPlayerElement_MediaPlayerElement_Sources`
  - `MediaPlayerElement_Using_3gp_UITests_Shared_Windows_UI_Xaml_Controls_MediaPlayerElement_MediaPlayerElement_3gp_Extension`
  - `MediaPlayerElement_Using_avi_UITests_Shared_Windows_UI_Xaml_Controls_MediaPlayerElement_MediaPlayerElement_Avi_Extension`
  - `MediaPlayerElement_Using_flv_UITests_Shared_Windows_UI_Xaml_Controls_MediaPlayerElement_MediaPlayerElement_Flv_Extension`
  - `MediaPlayerElement_Using_mkv_UITests_Shared_Windows_UI_Xaml_Controls_MediaPlayerElement_MediaPlayerElement_Mkv_Extension`
  - `MediaPlayerElement_Using_mov_UITests_Shared_Windows_UI_Xaml_Controls_MediaPlayerElement_MediaPlayerElement_Mov_Extension`
  - `MediaPlayerElement_Using_mp3_Audio_only_UITests_Shared_Windows_UI_Xaml_Controls_MediaPlayerElement_MediaPlayerElement_Mp3_Extension`
  - `MediaPlayerElement_Using_ogg_UITests_Shared_Windows_UI_Xaml_Controls_MediaPlayerElement_MediaPlayerElement_Ogg_Extension`
  - `PipsPager_MUXControlsTestApp_PipsPagerExamples_MUXControlsTestApp_PipsPagerExamples`
  - `ListView_SelectedItems_SamplesApp_Windows_UI_Xaml_Controls_ListView_ListViewSelectedItems`
  - `MediaPlayerElement_MediaPlayerElement_Stretch_Fill_UITests_Shared_Windows_UI_Xaml_Controls_MediaPlayerElement_MediaPlayerElement_Stretch_Fill`
  - `MediaPlayerElement_Mini_player_UITests_Shared_Windows_UI_Xaml_Controls_MediaPlayerElement_MediaPlayerElement_Minimal`
  - `Automation_UITests_Shared_Windows_UI_Xaml_Automation_AutomationProperties_Name_UITests_Shared_Windows_UI_Xaml_Automation_AutomationProperties_Name`
  - `TabView_UITests_Microsoft_UI_Xaml_Controls_TabViewTests_TabViewPage_UITests_Microsoft_UI_Xaml_Controls_TabViewTests_TabViewPage`
  - `When_ExtendedSelectionWithoutItemClick_Then_PointersEvents_UITests_Windows_UI_Xaml_Controls_ListView_ListView_Selection_Pointers`
  - `Progress_UITests_Microsoft_UI_Xaml_Controls_ProgressRing_WinUIProgressRingPage_UITests_Microsoft_UI_Xaml_Controls_ProgressRing_WinUIProgressRingPage`
  </details>

* `wasm-automated-net7.0-UWP-RuntimeTests`: **0** changed over 1

* `wasm-automated-net7.0-WinUI-RuntimeTests`: **0** changed over 1

</details>

jeromelaban commented:
@mergifyio backport release/stable/5.0
